docs/api-docs/building-storefronts/nextjs-commerce-guide.md

```diff
@@ -51,19 +51,21 @@ To get started with Next.js Commerce, you'll need to deploy a live version direc
 
    **Note:** If you have an existing Vercel account, you can sign in using those credentials.
 
-   ![Create Vercel account](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-02.png)
+   ![Create Vercel account](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_get_started.png)
 
-   ![Log in with existing Vercel account](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-03.png)
+3. After signing in, a **Create Git Repository** dialogue will appear with a dropdown to select your Git Scope, and a **Repository Name** text field. You may change the name of the project from the default or leave it as-is. Click **Create** to proceed.
 
-3. After signing in, a **Clone Git Repository** dialogue will appear with a link to the Next.js Commerce repository, a dropdown to select your Vercel account, and a **Project Name** text field. You may change the name of the project from the default or leave it as-is. Click **Continue** to proceed.
+   ![Create Git repository](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_repo.png)
 
-   ![Clone Git repository](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-04.png)
+4. The next dialogue asks if you would like to create a team or use your personal account. You may click on **Skip** if you don't want to create a Vercel Team.
 
-4. In the **Install Integrations** dialogue box, click **Install** next to the BigCommerce integration to connect your BigCommerce store to your Vercel project.
+   ![Create Vercel team](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_team.png)
 
-   ![Install BigCommerce integration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-05.png)
+5. In the **Add Integrations** dialogue box, click **Add** next to the BigCommerce integration to connect your BigCommerce store to your Vercel project.
```

Comment:
```suggestion
5. In the **Add Integrations** dialogue box, click **Add** to connect your BigCommerce store to your Vercel project.
```
Google developer documentation style guide advises against using directional language to orient readers.